azure-pipelines.ci.yml

```diff
@@ -1 +1,9 @@
-# initial commit
\ No newline at end of file
+pr: none
+
+trigger:
+  - master
+
+extends:
+  template: .devops/templates/pr-ci.yml
+  parameters:
+    isPR: true
```

Comment:
the `isPR` value is `true` for both PR and master pipelines is that intentional ?

Reply:
no 🤦‍♀️  fixed now